### PR TITLE
Add CFP related page content

### PR DESCRIPTION
**Pull request by mattwang44**
## Types of Changes

- [ ] **Bugfix**
- [x] **New feature**
- [ ] **Refactoring**
- [ ] **Breaking change** (any change that would cause existing functionality to not work as expected)
- [ ] **Documentation Update**
- [ ] **Other (please describe)**

## Description

- Add CFP related content
- The `*.i18n.js` in #8 have complex structures and have been simplified in this PR.

## More Information

- page content is provided by program team
	- [en-us version](https://hackmd.io/@pycontw/H1cdtGaqw)
	- [zh-hant version](https://hackmd.io/@pycontw/B1pbilG9D)

### CODE REVIEW

File: components/core/header/Header.vue [hunk 1]
```diff
@@ -28,6 +25,11 @@ export default {
         NavBar,
         TextButton,
     },
+    methods: {
+        getMyPyConUrl() {
+            return `https://tw.pycon.org/2021/${this.$i18n.locale}/dashboard/`
+        },
+    },
```
Comment (flynnhou) [inline components/core/header/Header.vue:32]: Consider using the `computed` property for consistency

File: pages/speaking/cfp.vue [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+            v-for="(date, i) in $t('importantDates.1')"
+            :key="i"
+            :path="`importantDates.1.${i}`"
+            tag="p"
+            class="my-1"
+        >
+            <template #AoE>
+                <ext-link
+                    href="https://www.timeanddate.com/worldclock/converter.html?iso=20210427T115900&p1=tz_aoe&p2=241&p3=1440"
+                    :highlight="true"
```
Comment (flynnhou) [inline pages/speaking/cfp.vue:18]: Remove unecessary `true`

```suggestion
                    highlight
```

File: pages/speaking/cfp.vue [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+        <i18n
+            v-for="(date, i) in $t('howToSubmit.1')"
+            :key="i"
+            :path="`howToSubmit.1.${i}`"
+            tag="p"
+            class="my-6"
+        >
+            <template #signUp>
+                <ext-link
+                    :href="proposalSystemUrlPrefix + '/accounts/login/'"
```
Comment (flynnhou) [inline pages/speaking/cfp.vue:60]: Consider using a method

File: pages/speaking/cfp.vue [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+                <ext-link
+                    href="https://pyfound.blogspot.com/2017/10/psfs-october-board-meeting.html"
+                    :highlight="true"
+                    >Everyone Contributes Policy</ext-link
+                >
+            </template>
+        </i18n>
+
+        <!-- How to Submit -->
+        <i18n path="howToSubmit.0" tag="h2" class="text-2xl my-8"></i18n>
```
Comment (flynnhou) [inline pages/speaking/cfp.vue:50]: Consider making the margin top greater than the margin bottom to distinguish the different groups of contents. This rule should apply to all top-level titles.

File: pages/speaking/cfp.vue [hunk 5]
```diff
@@ -0,0 +1,216 @@
...
+                    :href="proposalSystemUrlPrefix + '/dashboard'"
+                    :highlight="true"
+                    >My PyCon</ext-link
+                >
+            </template>
+        </i18n>
+
+        <!-- Guidelines -->
+        <i18n path="guidelines-title" tag="h2" class="text-2xl my-8"></i18n>
+        <i18n path="guidelines-talks.0" tag="h3" class="text-xl my-8"></i18n>
```
Comment (flynnhou) [inline pages/speaking/cfp.vue:79]: The `h3`s should have smaller margins

File: components/core/links/ExtLink.vue [hunk 6]
```diff
@@ -3,7 +3,7 @@
         :href="href"
         rel="noopener noreferrer"
         target="_blank"
-        class="outline-none"
+        :class="['outline-none', highlight ? 'highlight' : '']"
```
[the remark this answers is not in the document]
Comment (mattwang44) [reply components/core/links/ExtLink.vue]: updated

File: components/core/links/LocaleLink.vue [hunk 7]
```diff
@@ -1,5 +1,5 @@
 <template>
-    <nuxt-link :to="localePath(to)">
+    <nuxt-link :to="localePath(to)" :class="highlight ? 'highlight' : ''">
```
[the remark this answers is not in the document]
Comment (mattwang44) [reply components/core/links/LocaleLink.vue]: updated

File: components/core/header/Header.vue (hunk 1 above)
Comment (mattwang44) [reply components/core/header/Header.vue]: updated

File: pages/speaking/cfp.vue (hunk 2 above)
Comment (mattwang44) [reply pages/speaking/cfp.vue]: updated

File: pages/speaking/cfp.vue (hunk 3 above)
Comment (mattwang44) [reply pages/speaking/cfp.vue]: also adopt `computed` for consistency

File: pages/speaking/cfp.vue (hunk 5 above)
Comment (mattwang44) [reply pages/speaking/cfp.vue]: updated

File: pages/speaking/cfp.vue (hunk 4 above)
Comment (mattwang44) [reply pages/speaking/cfp.vue]: updated

**Review by flynnhou — APPROVED** [review]
lgtm

**Pull request merged**